models/ac_generator.h

```diff
@@ -128,12 +164,20 @@ class ac_generator : public Node
     double A_11_;
   };
 
+  double
+  get_I_() const
+  {
+    return S_.I_;
```

Comment:
@heplesser : I am not sure this would work properly (without additional changes). The problem would be before t_start and after t_stop, where S_.y_1_ is not being set to zero. 

The option might be to explicitly set S_.y_1_ to zero outside the required time period. But then we have the problem of P_.offset_, which will always be reflected in the recorded current. Unless we put in a check for time in get_I_().